src/vs/workbench/services/progress/browser/progressService.ts

```diff
@@ -7,7 +7,7 @@ import 'vs/css!./media/progressService';
 
 import { localize } from 'vs/nls';
 import { IDisposable, dispose, DisposableStore, MutableDisposable, Disposable } from 'vs/base/common/lifecycle';
-import { IProgressService, IProgressOptions, IProgressStep, ProgressLocation, IProgress, Progress, IProgressCompositeOptions, IProgressNotificationOptions, IProgressRunner, IProgressIndicator } from 'vs/platform/progress/common/progress';
+import { IProgressService, IProgressOptions, IProgressStep, ProgressLocation, IProgress, Progress, IProgressCompositeOptions, IProgressNotificationOptions, IProgressRunner, IProgressIndicator, IProgressStatusBarOptions as IProgressWindowOptions } from 'vs/platform/progress/common/progress';
```

Comment:
@alexr00 I would suggest to rename the actual type instead of the import

Reply:
Agh, my bad. I did a refactor rename and assumed that it did the right thing.